app/controllers/import_controller.rb

```diff
@@ -38,108 +46,139 @@ def import
 
     # Load and parse patient import excel
     begin
-      xlsx = Roo::Excelx.new(params[:file].tempfile.path, file_warning: :ignore)
-      validate_headers(format, xlsx.sheet(0).row(1))
-      raise ValidationError.new('File must contain at least one monitoree to import', 2) if xlsx.sheet(0).last_row < 2
-      raise ValidationError.new('Please limit each import to 1000 monitorees.', 1000) if xlsx.sheet(0).last_row > 1001
+      if format == :epix
+        ext = File.extname(params[:file].tempfile.path)
+        raise FileFormatError.new('File format must be .csv', ext) unless ext == '.csv'
+
+        csv = CSV.parse(File.read(params[:file].tempfile.path), headers: true)
+        validate_headers(format, csv.headers)
+      else
+        xlsx = Roo::Excelx.new(params[:file].tempfile.path, file_warning: :ignore)
+        validate_headers(format, xlsx.sheet(0).row(1))
+      end
+
+      num_rows = format == :epix ? csv.length : xlsx.sheet(0).last_row
+      raise ValidationError.new('File must contain at least one monitoree to import', 2) if num_rows < 2
```

Comment:
I'm noticing that if I edit a csv file in excel and delete the contents, it actually ends up looking something like this: 
![image](https://user-images.githubusercontent.com/52747882/119835917-47615180-becf-11eb-9c3a-0f445b1fde11.png)

which is parsed in as a csv file with more than 2 rows, even though all the rows are empty. So this validation error does not get hit, even though you are importing what looks (in excel) like an empty file. I'm not sure if this is a big enough deal to address, but wanted to point it out in case you thought it was.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/884/commits/0792f6104ae4af0333091eb76fb21a3418b3e5bd#diff-ee31e076a80cc9f48da0394c0cb066d59bf2f8c68f68eb62cb5119d01712d90eR53